packages/leancode_lint/lib/analysis_options.yaml

```diff
@@ -186,8 +186,6 @@ linter:
     # Included in lints recommended
     # - library_prefixes
 
-    - library_private_types_in_public_api
```

Comment:
Comment those rules out instead of removing them completely. I will be easier to track new lints being added to dart in the future

Reply:
Eh, now I realized that actually all disabled lints are here commented out. Yup, let's leave it then, but without the comment on the annoyance :)

Reply:
The rabbis reached an agreement.